### PR TITLE
Add astroquery

### PR DESCRIPTION
**Pull request by pkgw**
Trialing whether we can do this without having to package up astropy_helpers separately.

### CODE REVIEW

**Comment by pkgw** [comment]
CC @guillochon @astrofrog @mwcraig.

File: recipes/astroquery/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: $PYTHON setup.py install --offline --no-git --single-version-externally-managed --record=files.txt
```
Comment (mwcraig) [inline recipes/astroquery/meta.yaml]: Please change `$PYTHON` to `python` (this will fix the Windows fail).

Comment (pkgw) [inline recipes/astroquery/meta.yaml]: And I thought I was being so tidy! Fixed.

**Comment by pkgw** [comment]
Should be good to merge if there are no further tweaks to be made (CC @conda-forge/core).

**Pull request merged**